Tutorials/Moving_robot/moving_robot.md

```diff
@@ -0,0 +1,139 @@
+# Moving our robot
+
+In this tutorial we will learn how to move our robot. We will use the robot we built in [Build your own robot](SDF_tutorial_link) tutorial. You can download the robot from [here](car_world.sdf).
```

Comment:
This link is not working. It should be `car_demo`, and the directory is different from this one.